kubernetes/schema_metadata.go

```diff
@@ -79,6 +79,11 @@ func metadataSchema(objectName string, generatableName bool) *schema.Schema {
 	}
 }
 
+func metadataSchemaForceNew(s *schema.Schema) *schema.Schema {
+	s.ForceNew = true
```

Comment:
Something else you could do here is use the [CustomizeDiff](https://www.terraform.io/docs/extend/resources/customizing-differences.html) feature and use `ForceNewIfChange("metdata", ... `

I'm not sure which is better tbh.

Reply:
I don't think that the underscoring thing is necessary at all for function arguments.
If I'm not mistaken, Go doesn't account the function arguments as "unused variables" when they aren't actually user.

Did a quick test here: https://play.golang.com/p/hRJXvvbx_GJ

Reply:
Thanks for the suggestion. Since I'm omitting the `Update` function for this resource, all attributes need to have `ForceNew` set. Otherwise the SDK throws an error. I'm not sure that a `CustomizeDiff` function would satisfy that requirement. Plus it seems to be more of a standard practice to specify the `ForceNew` at the attribute level when possible. `CustomizeDiff` is most useful for cases where you need more complex logic to determine when an attribute should use `ForceNew`.